_data/menu.json

```diff
@@ -1,15 +1,23 @@
 [
-  { "title": "About", "url": "/about/", "subpages": [
-    { "title": "Governance", "url": "/governance/" }
-  ] },
+  {
+    "title": "Get Involved",
+    "url": "#",
+    "subpages": [
+      { "title": "Contributing", "url": "/contributing/" },
+      { "title": "Documentation", "url": "/documentation/" },
+      { "title": "Code of Conduct", "url": "/coc/" }
+    ]
+  },
   { "title": "Download", "url": "/download/" },
-  { "title": "Get Involved", "url": "/get-involved/", "subpages": [
-    { "title": "Contributing", "url": "/contributing/" },
-    { "title": "Documentation", "url": "/documentation/" },
-    { "title": "Code of Conduct", "url": "/coc/" }
-  ] },
-  { "title": "Sponsorship", "url": "/sponsorship/", "subpages": [
-    { "title": "Acknowledgements", "url": "/acknowledgements/" }
-  ] },
-  { "title": "Blog", "url": "/blog/" }
+  { "title": "Blog", "url": "/blog/" },
+  {
+    "title": "About",
+    "url": "#",
+    "subpages": [
+      { "title": "About Us", "url": "/about/" },
```

Comment:
i think “About Servo” would make more sense here, wdyt?

Reply:
indeed. thanks for the review :)
Fixed in [35ece43](https://github.com/servo/servo.org/pull/107/commits/35ece439131fbdfb7e9b23351aed30ed26be3c58)